### DISCUSSION TITLE
Public build test 03

### DISCUSSION THREAD

**Discussion by prayag17** [prayag17/Blink#77] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/prayag17/JellyPlayer/releases/tag/v0.0.1-publictest03'>Public build test 03</a>.</em>

**Comment by sabino-pereira** [comment]
I am using the appimage version and getting stuck on a blank screen. I think clearing the data might solve the issue as I did not  specify the port when giving the server address.  Can you tell me  where appimage stores its data?

**Reply by prayag17** [reply]
It should be inside appdata/roaming eq of macos in folder named com.jellyplayer.prayag17

**Reply by exalented** [reply]
Seeing this same issue on Arch Linux. Debugging, it shows "/Users/Me" is 401 Unauthorized after I login. Happens with different users on the jellyfin server. I've tried http(s) and I get the same result.

**Reply by prayag17** [reply]
It might show that at first but then it will continue to function as is, publictest are buggy so if you want to see the app on action try running it as mentioned in readme

**Reply by sabino-pereira** [reply]
Where would this folder be located in linux?

**Comment by not-toto** [comment]
Is it possible to get rid of the title bar? It makes the app appear ugly. Also, the hamburger menu does nothing, and the home button from the bottom left might make more sense up there.

**Reply by prayag17** [reply]
The sidemenu is still wip that's why the hamburger doesn't work
